lib/MultiCompiler.js

```diff
@@ -15,8 +16,10 @@ module.exports = class MultiCompiler extends Tapable {
 		super();
 		this.hooks = {
 			done: new SyncHook(["stats"]),
-			invalid: new SyncHook([]),
-			watchClose: new SyncHook([])
+			invalid: new MultiHook(compilers.map(c => c.hooks.invalid)),
+			run: new MultiHook(compilers.map(c => c.hooks.run)),
+			watchClose: new SyncHook([]),
```

Comment:
This one cannot be converted to a `MultiHook` because of https://github.com/webpack/webpack/blob/next/lib/Compiler.js#L167